password-cli/password-cli.cabal

```diff
@@ -0,0 +1,38 @@
+cabal-version:  3.0
+
+name:           password-cli
+version:        0.1.0.0
+category:       CLI
+synopsis:       use password from your CLI
+description:    A simple CLI tool to interact with password
+homepage:       https://github.com/cdepillabout/password/tree/master/password-cli#readme
+bug-reports:    https://github.com/cdepillabout/password/issues
+author:         Dennis Gosnell, Felix Paulusma
+maintainer:     cdep.illabout@gmail.com, felix.paulusma@gmail.com
+copyright:      Copyright (c) Dennis Gosnell, 2019; Felix Paulusma, 2020
+license:        BSD-3-Clause
+license-file:   LICENSE
+build-type:     Simple
+extra-source-files:
+    README.md
+    ChangeLog.md
+
+source-repository head
+  type: git
+  location: https://github.com/cdepillabout/password
+
+
+executable password-cli
+  -- type: exitcode-stdio-1.0
+  main-is: Main.hs
+  other-modules:
+    Paths_password_cli
+  hs-source-dirs: app
+  ghc-options: -threaded -rtsopts -with-rtsopts=-N -Wall -Wcompat -Widentities -Wincomplete-record-updates -Wincomplete-uni-patterns -Wpartial-fields -Wredundant-constraints
+  build-depends:
+      base
+    , password ^>= 3.0
```

Comment:
Since `password-cli` uses the `Bcrypt.defaultParams`, this needs to be `>= 3.0.3.0 && < 3.1`, right?

`^>= 3.0` also includes `>= 3.0.0.0 < 3.0.3.0`, right?